test/fw/ptl/utils/jobs/eatcpu.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python3
```

Comment:
If we use this shebang: #!/usr/bin/python3, we have to create a softlink for this. It might break something on system as some applications will be dependent on python2. 
While #!/usr/bin/env python will run whichever python is installed on the system.

Reply:
Well, this script is supposed to be used for testing purposes, at least chiefly. We use "#!/usr/bin/env python3" in all of our scripts now. So, I think we should be consistent unless there's an issue with using "#!/usr/bin/env python3"

Reply:
changed the shebang to #!/usr/bin/env python3 as with earlier shebang "/usr/bin/env python" it was failing in debian and ubuntu18 (Travis)